runelite-client/src/main/java/net/runelite/client/game/ItemMapping.java

```diff
@@ -252,6 +263,10 @@ public enum ItemMapping
 	ITEM_TUMEKENS_SHADOW(TUMEKENS_SHADOW_UNCHARGED, TUMEKENS_SHADOW),
 	ITEM_PHARAOHS_SCEPTRE(PHARAOHS_SCEPTRE_UNCHARGED, true, 1L, PHARAOHS_SCEPTRE),
 	ITEM_VENATOR_BOW(VENATOR_BOW_UNCHARGED, VENATOR_BOW),
+	ITEM_ECHO_BOOTS(GUARDIAN_BOOTS, ECHO_BOOTS),
+
+	// Demonbane weapons
+	ITEM_TORMENTED_SYNAPSE(TORMENTED_SYNAPSE, EMBERLIGHT, SCORCHING_BOW, PURGING_STAFF),
```

Comment:
I'm not sure whether it would be helpful to most players to display this price on these items, since the only way to get the synapse back from these is to lose it in pvp combat.

Reply:
Makes sense, I wasn't positive those item changes would pass, but figured I'd add them in case someone deemed them appropriate.